service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MessageJudgeController.java

```diff
@@ -82,12 +83,21 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
     @PostMapping("/submitted")
     public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
         CaseData caseData = getCaseData(callbackRequest.getCaseDetails());
-        JudicialMessage judicialMessage = caseData.getJudicialMessages().get(0).getValue();
 
-        if (judicialMessage.isFirstMessage()) {
-            publishEvent(new NewJudicialMessageEvent(caseData, judicialMessage));
-        } else {
-            publishEvent(new JudicialMessageReplyEvent(caseData, judicialMessage));
+        List<Element<JudicialMessage>> judicialMessages = caseData.getJudicialMessages();
+        if (!CollectionUtils.isEmpty(caseData.getClosedJudicialMessages())) {
+            judicialMessages.addAll(caseData.getClosedJudicialMessages());
+        }
+
+        judicialMessages = messageJudgeService.sortJudicialMessages(judicialMessages);
+        JudicialMessage judicialMessage = judicialMessages.get(0).getValue();
+
+        if (OPEN.equals(judicialMessage.getStatus())) {
+            if (judicialMessage.isFirstMessage()) {
```

Comment:
probably not part of the story since we want to release on Monday, but would be nice to add coverage to the new email notification tests when we run through the code

Reply:
created a new PR for email template tetsing https://github.com/hmcts/fpl-ccd-configuration/pull/2088